estimator/serenity_estimator.cpp

```diff
@@ -35,14 +35,22 @@ class SerenityEstimatorProcess :
 
   Future<Resources> _oversubscribable(
       const Future<ResourceUsage>& _resourceUsage) {
-    Try<Resources> ret = this->pipeline->run(_resourceUsage.get());
+    // TODO(bplotka): Decide if need such simple algorithm.
+    Resources allocatedRevocable;
```

Comment:
We really should talk about it.. It's not clearly understandable how to behave on such situation. Our pipeline estimate resources without BE tasks included. We are filtering out them and take only PR and then return slack. It does not matter how many tasks BE were scheduled on that. Here we are just decreasing estimate using already allocated revocables, because slave understands that differently (https://github.com/apache/mesos/blob/master/src/slave/slave.cpp#L4251)